### ISSUE TITLE
Editing "Bug"

### ISSUE THREAD

**Issue by JustMe77** [Pottus/Texture-Studio#44]
Hi, I don't know if this is a bug, but it's pretty weird and don't let me to map accurate if I use group commands.

See here.

If I clone an object with /clone and use /editobject, it's normal, but if clone in a group and use /editgroup, it's weird, don't know how to explain.

See here

https://www.youtube.com/watch?v=5x7lj0N6znw&feature=youtu.be

**Comment by Crayder** [comment]
I don't see any bug. Editing a single object will edit it on it's axis. Editing a group will edit on the origin axis, (0, 0, 0).

**Comment by JustMe77** [comment]
Here's a second video.

The object's are in completely different positions, even I am doing the same thing with 2 different commands. I can't map accurate like this..

https://www.youtube.com/watch?v=K7g1EqC2dK0&feature=youtu.be

**Comment by Crayder** [comment]
![image](https://cloud.githubusercontent.com/assets/9919010/18004127/f510be88-6b5f-11e6-8b85-3f6e0f1c596c.png)

**Comment by JustMe77** [comment]
try now

**Comment by Crayder** [comment]
Seen it, the same thing I said in my last comment is what is happening. It's not a bug, your house is just rotated. `/editobject` edits along the objects' axis while `/ox`, `/oy`, `/oz`, `/editgroup` and other commands all edit on the origin axis.

**Comment by JustMe77** [comment]
Well, I thought maybe you could create a command / function so I can edit the objects with the matching rotation, if possible.

If not, then feel free to close this issue

**Comment by Crayder** [comment]
Yeah, it's possible. Just have to make a rotated pivot object for the group editor and a rotated vector function for the /o* commands.

**Comment by JustMe77** [comment]
Nice, hope it's easy and will be done in the future :)
